api/envoy/config/filter/http/aws_lambda/v2/aws_lambda.proto

```diff
@@ -40,6 +40,10 @@ message AWSLambdaPerRoute {
   // used to process response data.
   // This config cannot be configured simultaneously with unwrap_as_alb
```

Comment:
nit: I imagine that the preferred approach is to no longer use unwrap_as_alb and instead defined a response transformer config to handle that. Can we flag the unwrap_as_alb API as deprecated and include a comment in the proto outlining the preferred way to configure that behavior?

Reply:
Updated that comment here: cb39433a0958a4bec1cae76b686fe1fbbe5ec390
Let me know what you think!